GNUmakefile

```diff
@@ -64,6 +65,9 @@ format:
 	@echo "--> Running go fmt"
 	@go fmt $(PACKAGES)
 
+golint:
+	golint
```

Comment:
This tag will only run `golint` tool in root folder , this issue is for running `golint` in all pkg's inside the openebs/maya repo.

Reply:
It will be better to have `gometalinter`  which will be aggregate and normalise the output of a whole bunch of Go linters, not just `golint`